Sources/Asset.swift

```diff
@@ -12,6 +12,40 @@ import ObjectMapper
 /// An asset represents a media file in Contentful
 public class Asset: Resource, LocalizedResource {
 
+    /// URL of the media file associated with this asset. Optional for compatibility with `select` operator queries.
+    public var urlString: String?
```

Comment:
Hey, do you have any plans on also supporting the `upload` url field? (i.e. the one used while an asset is processing?)